native/src/lib.rs

```diff
@@ -14,12 +14,12 @@ use neon::js::class::{JsClass, Class};
 use fst::{Set, SetBuilder};
 
 
-trait CheckArgument<'a> {
-  fn check_argument<V: Value>(&mut self, i: i32) -> JsResult<'a, V>;
+trait CheckArgument {
+  fn check_argument<V: Value>(&mut self, i: i32) -> JsResult<V>;
 }
 
-impl<'a, T: This> CheckArgument<'a> for FunctionCall<'a, T> {
-  fn check_argument<V: Value>(&mut self, i: i32) -> JsResult<'a, V> {
+impl<'a, T: This> CheckArgument for FunctionCall<'a, T> {
+  fn check_argument<V: Value>(&mut self, i: i32) -> JsResult<V> {
```

Comment:
I wish I knew exactly why removing these lifetime annotations worked. I don't.  I just removed ALL annotations and added them back one by one because the compiler yelled at me.